File: app/src/main/java/com/udacity/maluleque/meutako/model/Category.java
package com.udacity.maluleque.meutako.model;

import com.google.firebase.firestore.IgnoreExtraProperties;

@IgnoreExtraProperties
public class Category {

    private String name;
    private String type;

    public Category() {
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: app/src/main/java/com/bernardo/maluleque/shibaba/model/Category.kt
package com.bernardo.maluleque.shibaba.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.firebase.firestore.IgnoreExtraProperties

@IgnoreExtraProperties
@Entity
data class Category(@PrimaryKey(autoGenerate = true) val uid: Int = 0,
                    @ColumnInfo(name = "name") val type: CategoryType,
                    @ColumnInfo(name = "type") val name: String

)